### ISSUE TITLE
bug(query): EC2 Not EBS Optimized False Positive

### ISSUE THREAD

**Issue by joaoReigota1** [Checkmarx/kics#4968]
When using a t3.nano instance type in Cloudformation, and not specifying EbsOptimized, KICS is reporting that EC2 is not EBS Optimized, although t3.nano is already EBS Optimized by default and this property will not have any effect.

For more information relating to this issue: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-optimized.html#ebs-optimization-support

Keep in mind that some instances types do not support EBS Optimization and a filter should be added to this as well

### Expected Behavior
``` yaml
Type: AWS::EC2::Instance
    Properties:
      InstanceType: t3.nano
```
No results related to EBS Optimization

### Actual Behavior
``` yaml
Type: AWS::EC2::Instance
    Properties:
      InstanceType: t3.nano
```
EC2 Not EBS Optimized, Severity: INFO, Results: 1

### Steps to Reproduce the Problem

(Command line arguments and flags used)

1.  build a template file with a t3.nano AWS::EC2::Instance resource
2.  scan with KICS

### Specifications

-   Version: v1.5.3
-   Platform: Cloudformation
-   Subsystem: linux
